clients/spark/build.sbt

```diff
@@ -1,102 +1,54 @@
-import build.BuildType
-
-lazy val baseName = "lakefs-spark"
 lazy val projectVersion = "0.10.0"
 lazy val hadoopVersion = "3.2.1"
 
-ThisBuild / isSnapshot := false
+ThisBuild / isSnapshot := true
```

Comment:
Not expert on sbt - but do we need to mark this one always as `true` it is not set based on how we publish?

Reply:
It was just an experiment, good catch